airbyte-integrations/connectors/source-declarative-manifest/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+data:
+  ab_internal:
+    ql: 100
+    sl: 100
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.2.0@sha256:c22a9d97464b69d6ef01898edf3f8612dc11614f05a84984451dde195f337db9
+  connectorSubtype: api
+  connectorType: source
+  definitionId: 64a2f99c-542f-4af8-9a6f-355f1217b436
+  # This version should not be updated manually - it is updated by the CDK release workflow.
+  dockerImageTag: 0.78.6
+  dockerRepository: airbyte/source-declarative-manifest
+  documentationUrl: https://docs.airbyte.com/integrations/sources/low-code
+  githubIssueLabel: source-declarative-manifest
+  license: MIT
+  name: Low-Code Source
+  registries:
+    cloud:
+      enabled: false
+    oss:
+      enabled: false
+  releaseDate: 2023-03-01
+  releaseStage: alpha
+  supportLevel: community
```

Comment:
Yeah, I think that's an issue with the "supportLevel"'s meaning in any case - "we support it" does not mean it's certified by our own internal standards. I agree it doesn't matter at the moment since we don't actually show this source in the docs/UI, but we should probably revisit this.